database/migrations/20200209204804_application_submissions.js

```diff
@@ -0,0 +1,28 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("application_submissions", function(table) {
+    table.increments("id");
+    table
+      .integer("skilled_impact_request_id")
+      .references("id")
+      .inTable("skilled_impact_requests")
+      .notNullable()
+      .unsigned()
+      .onDelete("RESTRICT")
+      .onUpdate("CASCADE");
+    table
+      .integer("user_id")
+      .references("id")
+      .inTable("users")
+      .notNullable()
+      .unsigned()
+      .onDelete("RESTRICT")
+      .onUpdate("CASCADE");
+    table.enu("decision", ["ACCEPTED", "PENDING", "DENIED"]).defaultTo("PENDING");
```

Comment:
Can you make this a native enum for better performance please? See #100 migration

Reply:
@dacruz21 just looked into native enums and it seems like they're immutable (see [here](https://docs.hasura.io/1.0/graphql/manual/schema/enums.html#using-native-postgres-enum-types))? That might cause some issues with the skills enum in #100 since we might need to add/remove values in the future.

Reply:
They're only immutable *within transactions*. I usually get around this by just creating a transaction-less migration when I need to add/remove from an enum. Not ideal but (IMO) worth it for the performance benefit. Lmk if you disagree

Reply:
What're the performance benefits? I can't really find much documentation about native enums but I could see it being difficult to maintain if someone doesn't know about the transactions caveat (I definitely didn't).